modules/core/src/test/java/org/apache/ignite/internal/processors/cache/transactions/TxPartitionCounterStateConsistencyTest.java

```diff
@@ -963,27 +965,29 @@ public void testPartitionConsistencyDuringRebalanceAndConcurrentUpdates_LateAffi
         CountDownLatch resumeDiscoSndLatch = new CountDownLatch(1);
 
         BlockTcpDiscoverySpi crdDiscoSpi = (BlockTcpDiscoverySpi)grid(0).configuration().getDiscoverySpi();
-        CyclicBarrier sync = new CyclicBarrier(2);
+        CountDownLatch sync = new CountDownLatch(1);
 
         crdDiscoSpi.setClosure((node, msg) -> {
             if (msg instanceof CacheAffinityChangeMessage) {
-                U.awaitQuiet(sync);
-                U.awaitQuiet(resumeDiscoSndLatch);
+                sync.countDown();
+
+                try {
+                    if (!U.await(resumeDiscoSndLatch, 10, TimeUnit.SECONDS))
+                        fail("So it did not wait the latch yet.");
```

Comment:
Let's change this message as follows: `Failed to wait for GridNearLockRequest from client node`.